src/settings-ui/Settings.UI/Controls/SettingsPageControl/SettingsPageControl.xaml

```diff
@@ -56,12 +59,17 @@
                     </Border>
 
                     <StackPanel Grid.Column="1">
-                        <TextBlock x:Name="AboutDescription"
-                                   Text="{x:Bind ModuleDescription}"
-                                   Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                                   TextWrapping="Wrap"/>
+                        <TextBlock
+                            x:Name="AboutDescription"
+                            Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                            Text="{x:Bind ModuleDescription}"
+                            TextWrapping="Wrap" />
 
-                        <ItemsControl x:Name="PrimaryLinksControl" ItemsSource="{x:Bind PrimaryLinks}" IsTabStop="False" Margin="0,8,0,0">
+                        <ItemsControl
+                            x:Name="PrimaryLinksControl"
+                            Margin="0,8,0,0"
+                            IsTabStop="False"
+                            ItemsSource="{x:Bind PrimaryLinks}">
                             <ItemsControl.ItemTemplate>
                                 <DataTemplate x:DataType="local:PageLink">
                                     <HyperlinkButton NavigateUri="{x:Bind Link}" Style="{StaticResource TextButtonStyle}">
```

Comment:
Maybe it's me,but this (style) is not as pretty as it could be...
![image](https://user-images.githubusercontent.com/65828559/196541693-8a5cac1f-4f60-4133-8453-d78e1d1a0ee0.png)